src/modules/login/components/socialLine/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import {StyleSheet, Text, TouchableOpacity, View} from "react-native";
+import Ionicons from "@expo/vector-icons/Ionicons";
+import {Colors, Indents, Shadows, TextSize} from "../../../../constants/styles";
+import {Shadow} from "react-native-shadow-2";
+
+const BORDER_RADIUS = 20;
+const BUTTON_SIZE = 32;
+
+const SocialLine = () => {
```

Comment:
Why we need index file and separate folder for it?

Reply:
to make common component there :)